frontend/src/components/markdown/MathEditorDialog.tsx

```
@@ -0,0 +1,246 @@
+                      size="sm"
+                      variant={!isInline ? 'secondary' : 'ghost'}
+                      onClick={() => setIsInline(false)}
+                    >
+                      Block
+                    </Button>
+                  </div>
+                </div>
+                <Card className="h-32 flex items-center justify-center p-4 overflow-auto">
+                  {renderedMath ? (
+                    <div
+                      dangerouslySetInnerHTML={{ __html: renderedMath }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)